### PR TITLE
Make this package mip compatable.

### PR DESCRIPTION
**Pull request by Brenden2008**
Mip is the replacement for upip in the latest versions of Micropython. This pull request adds mip compatibility to this package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "urls": [
+        [
+            "uthingsboard/__init__.py",
+            "github:Brenden2008/thingsboard-micropython/uthingsboard/__init__.py"
```
Comment (manuargue) [inline package.json:5]: Please point it to this repo instead to your fork.

**Review by manuargue — CHANGES_REQUESTED** [review]
Hi, thanks for the pr! Honestly long time I haven’t used this, I’m glad it is still relevant :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        ],
+        [
+            "uthingsboard/client.py",
+            "github:Brenden2008/thingsboard-micropython/uthingsboard/client.py"
+        ]
+    ],
+    "deps": [
+    ],
+    "version": "1.0.0"
```
Comment (manuargue) [inline package.json]: Please use 0.2.0 as this was the last version published.

**Comment by Brenden2008** [comment]
Thanks for the feedback! I've corrected both issues.

**Review by manuargue — APPROVED** [review]

**Review by manuargue — CHANGES_REQUESTED** [review]
Oh one last request, please squash all the commits so that the change is atomic. Then I’m ok to merge. Thanks

**Comment by Brenden2008** [comment]
Ok I am going to be honest with you, I have no clue how to do that. So sorry, I am a bit new to git. I can open a new PR with all of the changes in one commit if you'd like?

**Comment by manuargue** [comment]
Sure np, I’ll close this then.

**Comment by Brenden2008** [comment]
Alright thanks.

**Pull request closed without merging**